src/query/plan/rewrite/index_lookup.hpp

```diff
@@ -171,6 +171,11 @@ class IndexLookupRewriter final : public HierarchicalLogicalOperatorVisitor {
     if (expand.common_.existing_node) {
       return true;
     }
+    if (expand.filter_lambda_.accumulated_path_symbol) {
+      // When accumulated path is used, we cannot use ST shortest path algorithm.
+      return false;
```

Comment:
so this applies for weighted shortest path as well as all shortest path?
do we lose performance on some queries because of this?

Reply:
Thank you! Added check for BFS. Performance should be affected in BFS because we will not use STShortestPath algo.